### PR TITLE
[SafetyKit] Added support for Xcode 15.2

### PR DESCRIPTION
**Pull request by SotoiGhost**
~As this framework is also available for macOS 13.0, let's wait for it to be added again within Xcode 14~

### CODE REVIEW

File: src/SafetyKit/SAEnums.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), Watch (9,0), NoTV]
+	[Native]
+	public enum SAEmergencyResponseManagerVoiceCallStatus : long
+	{
+		Dialing,
+		Active,
+		Disconnected,
+		Failed
```
Comment (rolfbjarne) [inline src/SafetyKit/SAEnums.cs:42]:
```suggestion
		Failed,
```

File: src/SafetyKit/SAEnums.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), Watch (9,0), NoTV]
+	[ErrorDomain ("SAErrorDomain")]
+	[Native]
+	public enum SAErrorCode : long
+	{
+		NotAuthorized = 1,
+		NotAllowed,
+		InvalidArgument,
+		OperationFailed
```
Comment (rolfbjarne) [inline src/SafetyKit/SAEnums.cs:53]:
```suggestion
		OperationFailed,
```

**Review by rolfbjarne — APPROVED** [review]

**Review by chamons — APPROVED** [review]

**Comment by rolfbjarne** [comment]
Looks like this doesn't build:

> build/mac/mobile/SafetyKit/SAEmergencyResponseManager.g.cs(116,63): error CS8604: Possible null reference argument for parameter 'item2' in 'Tuple<bool, NSError>.Tuple(bool item1, NSError item2)'.

**Comment by rolfbjarne** [comment]
Closing in favor of #20263.

**Pull request closed without merging**